lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -35,9 +36,100 @@ export type AccountInfo = {
     };
     nativeAccountId?: string;
     authorityType?: string;
+    tenantProfiles?: Map<string, TenantProfile>;
+};
+
+/**
+ * Account details that vary across tenants for the same user
+ *
+ * - tenantId               - Full tenant or organizational id that this account belongs to
```

Comment:
Doc comments for params should go above each individual field. You can leave the general description of the type here.